modules/moderation.py

```diff
@@ -339,11 +339,28 @@ async def _banning(self, ctx, users: commands.Greedy[ResolveUser], *, reason='-N
         failedBans = 0
         for user in users:
             userid = user if (type(user) is int) else user.id
-
             username = userid if (type(user) is int) else f'{str(user)}'
-            user = (
-                discord.Object(id=userid) if (type(user) is int) else user
-            )  # If not a user, manually contruct a user object
+
+            # If not a user, manually contruct a user object
+            user = discord.Object(id=userid) if (type(user) is int) else user
+
+            mod_role_pos = ctx.author.top_role.position
+            bot_member = await ctx.guild.fetch_member(self.bot.user.id)
```

Comment:
Might be better to do this one level up, to prevent duplicate API calls

Reply:
`guild.me`